validation-test/compiler_crashers_2_fixed/sr9199.swift

```diff
@@ -0,0 +1,57 @@
+// RUN: %target-swift-frontend -emit-ir -o %t.ll %s
```

Comment:
It speeds up the test slightly to not write to stdout. I could have picked /dev/null instead, I guess, but if something fails it might be useful to see if anything was output.